packer/scripts/centos/ppc64.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -eux
+
+yum update -y
+yum install -y ppc64-diag
```

Comment:
I would just move this to the ``openstack.sh`` script and only install it if ``uname -m `` is either ``ppc64`` or ``ppc64le``.